application/testing/CMakeLists.txt

```diff
@@ -1147,8 +1147,5 @@ if(WIN32 OR APPLE OR F3D_TESTING_ENABLE_GLX_TESTS) # Watch testing require GLX r
   endif()
   # Custom bash/pwsh test for testing watch option
   add_test (NAME f3d::TestWatch COMMAND ${_f3d_os_script_exec} ${CMAKE_CURRENT_SOURCE_DIR}/test_watch.${_f3d_os_script_ext} $<TARGET_FILE:f3d> ${F3D_SOURCE_DIR}/testing/data ${CMAKE_BINARY_DIR}/Testing/Temporary)
-  set_tests_properties(f3d::TestWatch PROPERTIES RUN_SERIAL TRUE)
-  if(F3D_COVERAGE) # When running test for coverage, adapt the script to use xdotool
-    set_tests_properties(f3d::TestWatch PROPERTIES ENVIRONMENT "CTEST_F3D_COVERAGE=1")
-  endif()
+  set_tests_properties(f3d::TestWatch PROPERTIES RUN_SERIAL TRUE TIMEOUT 60)
```

Comment:
thats a high timeout, any reason why ?

Reply:
It was 1 hour before :P

Reply:
ha you mean it was not defined and defaulted to 1h , ok!

Maybe make it lower ? I dont know how long this test takes on macOS

Reply:
I think it's fast. Around 7 seconds. I will lower it to 30s to be sure.